app/models/form/sales/pages/managing_organisation.rb

```diff
@@ -12,10 +12,20 @@ def questions
 
   def routed_to?(log, current_user)
     return false unless current_user
-    return false unless current_user.support?
-    return false unless FeatureToggle.sales_managing_organisation_enabled?
-    return false unless log.owning_organisation
 
-    log.owning_organisation.managing_agents.count >= 1
+    if form.start_year_after_2024?
+      organisation = current_user.support? ? log.owning_organisation : current_user.organisation
+
+      return false unless organisation
+      return false if log.owning_organisation != organisation && !organisation.holds_own_stock?
+      return true unless organisation.holds_own_stock?
+
+      organisation.managing_agents.count >= 1
+    else
+      return false unless current_user.support?
+      return false unless log.owning_organisation
+
+      log.owning_organisation.managing_agents.count >= 1
```

Comment:
just double checking - we can guarantee managing_agents includes absorbed orgs right?

Reply:
Do you mean displaying merged orgs as managing agents options? I think we would if there was a relationship, but should we in any other case?

Reply:
Yep that's all I was asking :)